src/deploy_tools/app_builder.py

```diff
@@ -96,3 +102,38 @@ def _create_shell_file(self, app: ShellApp, module: Module) -> None:
             executable=True,
             create_parents=True,
         )
+
+    def _create_binary_file(self, app: BinaryApp, module: Module) -> None:
+        """
+        Download a URL, validate it against its sha256, make it executable
+        and add it to PATH
+        """
+        binary_folder = self._build_layout.get_entrypoints_folder(
```

Comment:
I think stick to consistent folder naming. That could end up being 'bin_folder'...